QUser/src/java/ru/apertum/qsys/quser/Form.java

```diff
@@ -718,7 +718,7 @@ public void clickListPostponedInvite() {
 
                 setKeyRegim(KEYS_INVITED);
                 BindUtils.postNotifyChange(null, null, Form.this, "postponList");
-                BindUtils.postNotifyChange(null, null, Form.this, "customer");
+                BindUtils.postNotifyChange(null, null, Form.this, "citizen");
```

Comment:
There's no object named "citizen". I will leave it as customer only.
Check line 321 to 330. And I have no idea, where else the customer has been used.